hwilib/cli.py

```diff
@@ -26,7 +26,7 @@ def displayaddress_handler(args, client):
     return displayaddress(client, desc=args.desc, path=args.path, sh_wpkh=args.sh_wpkh, wpkh=args.wpkh)
 
 def enumerate_handler(args):
-    return enumerate(password=args.password)
+    return enumerate(password=args.password, device_type=args.device_type)
```

Comment:
I'm not sure that this is safe if device type is not specified.

Also, the device type may not necessarily be just the type, it could also be the model name in which the changes to `enumerate()` would not work.